extra-network/zerotier-one/autobuild/build

```
@@ -1,4 +1,9 @@
+abinfo "Substitute zerotier-one.service path..."
 sed -i 's/sbin/bin/' make-linux.mk debian/zerotier-one.service
```

2. We need to use absolute path to every file or directory, use `$SRCDIR` as the root directory:
```suggestion
sed -i 's/sbin/bin/' "$SRCDIR"/make-linux.mk "$SRCDIR"/debian/zerotier-one.service
```

3. And for `sed` operation, we suggest use it like this to make the purpose more clear to others:
```suggestion
sed -e 's/sbin/bin/' \
    -i "$SRCDIR"/make-linux.mk \
    -i "$SRCDIR"/debian/zerotier-one.service
```
（By the way, if a line exceeds 80 characters, it's recommended to wrap it at where is close to 80 characters.)
If necessary, you are supposed to use [patches](https://wiki.aosc.io/developer/packaging/autobuild3-manual/#the-patches-directory) instead of `sed` since the latter may cause silent failure.